mobilitypath_visualizer/test/test_mobility_path_visualizer.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019-2020 LEIDOS.
+ * Copyright (C) 2022 LEIDOS.
```

Comment:
Let's keep it as 2019-2022 and I think there are some other files (like package.xml) that still need to be edited as well.